packages/core/test/Form_test.js

```diff
@@ -616,10 +616,6 @@ describeRepeated("Form common", createFormComponent => {
       const { node } = createFormComponent({ schema });
 
       expect(node.querySelector("#root_children_0_name")).to.not.exist;
-
-      Simulate.click(node.querySelector(".array-item-add button"));
-
-      expect(node.querySelector("#root_children_0_name")).to.exist;
```

Comment:
Did these tests break?

Reply:
This one broke. But if you look at the schema in the unit test, I think it broke as expected. Should we still continue to support this scenario?

Reply:
Oh I see, this test is trying to load a circular ref. 👍  Good to remove these assertions then.